store/volume.go

```diff
@@ -4,9 +4,7 @@ package store
 // volumes stored in the store
```

Comment:
It's better to remove this file and the functions here all together. Having empty wrapper functions doesn't serve any purpose.